v2/cmd/voucher_server/README.md

```diff
@@ -56,14 +56,30 @@ Below are the configuration options for Voucher Server:
 | `server`             | `password`                   | A password hashed with the bcrypt algorithm, for use with the username.                               |
 | `ejson`              | `dir`                        | The path to the ejson keys directory.                                                                 |
 | `ejson`              | `secrets`                    | The path to the ejson secrets.                                                                        |
+| `sops`               | `file`                       | The path to the SOPS secrets.                                                                         |
 | `clair`              |  `address`                   | The hostname that Clair exists at. If "http://" or "https://" is omitted, this will default to HTTPS. |
 | `repository.[alias]` | `org-url`                    | The URL used to determine if a repository is owned by an organization.                                |
 | `required.[env]`     | (test name here)             | A test that is active when running "env" tests.                                                       |
+| `statsd`             | `backend`                    | The destination for reporting metrics, can be `statsd` for local aggregation, or `datadog`.           |
+| `statsd`             | `addr`                       | The UDP endpoint to use when `statsd.backend == "statsd"`.                                            |
+| `statsd`             | `tags`                       | List of tags in `key:value` format to apply to very metric. Example: `env:production`.                |
+| `statsd`             | `sample_rate`                | Configurable sample rate to limit metics overhead.                                                    |
```

Comment:
```suggestion
| `statsd`             | `addr`                       | The UDP endpoint to use when `statsd.backend == "statsd"`.                                            |
| `statsd`             | `tags`                       | List of tags in `key:value` format to apply to the metric. Example: `env:production`.                |
| `statsd`             | `sample_rate`                | Configurable sample rate to limit metrics overhead.                                                    |
```

Reply:
@dani-santos-code excellent catch on the typos 🤦 .

I tweaked the wording your first suggestion, because I think it's important to note that `tags` are sent with "every" metric emit by voucher, WDYT?

Reply:
oh, okie! yeah! sounds good!